workflow/controller/exec_control.go

```diff
@@ -116,7 +116,7 @@ func (woc *wfOperationCtx) killDaemonedChildren(ctx context.Context, nodeID stri
 				firstErr = err
 			}
 		}
-
+		childNode.Phase = wfv1.NodeSucceeded
```

Comment:
Set daemoned nodes to Succeeded when workflow ends

Reply:
test?

Reply:
Done